CHANGELOG.md

```diff
@@ -6,6 +6,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.8.0]
+### Added
+- Enhanced the build process by allowing build types to override the build name ([#22220] (https://github.com/MetaMask/metamask-extension/pull/22220))
+- Enhanced the Networks List with drag and drop functionality ([#21163] (https://github.com/MetaMask/metamask-extension/pull/21163))
+- Added a deprecation warning for the Aurora network to inform users about its upcoming removal ([#21933] (https://github.com/MetaMask/metamask-extension/pull/21933))
+- Added a survey prompt to the home screen for user feedback ([#21918] (https://github.com/MetaMask/metamask-extension/pull/21918))
+
+### Changed
+- Corrected a typo and improved the alignment of 'Terms of use' ([#22227] (https://github.com/MetaMask/metamask-extension/pull/22227))
+- Removed remaining references to the Aurora network ([#22184] (https://github.com/MetaMask/metamask-extension/pull/22184))
+- Changed the title of MetaMask pop-up window to 'MetaMask Dialog' ([#21680] (https://github.com/MetaMask/metamask-extension/pull/21680))
+- Refined the Max fee calculation in Smart Swaps to be no more than double the estimated gas fee ([#22127] (https://github.com/MetaMask/metamask-extension/pull/22127))
```

Comment:
Maybe "Refined the Max fee calculation in Smart Swaps to ensure it does not exceed twice the estimated gas fee" ?